homeassistant/components/mqtt/vacuum/schema_legacy.py

```diff
@@ -171,24 +189,43 @@ class MqttVacuum(MqttEntity, VacuumEntity):
     _entity_id_format = ENTITY_ID_FORMAT
     _attributes_extra_blocked = MQTT_LEGACY_VACUUM_ATTRIBUTES_BLOCKED
 
-    def __init__(self, hass, config, config_entry, discovery_data):
+    _encoding: str | None
+    _fan_speed_list: list[str]
```

Comment:
Are all these still needed with the new attributes?

Reply:
_fan_speed_list can be removed, the rest are not entity attributes.